lib/community/utils/post_card_action_helpers.dart

```diff
@@ -183,41 +169,9 @@ void showPostActionBottomModalSheet(BuildContext context, PostViewMedia postView
 }
 
 void onTapCommunityName(BuildContext context, int communityId) {
-  AccountBloc accountBloc = context.read<AccountBloc>();
-  AuthBloc authBloc = context.read<AuthBloc>();
-  ThunderBloc thunderBloc = context.read<ThunderBloc>();
-
-  Navigator.of(context).push(
-    SwipeablePageRoute(
-      canOnlySwipeFromEdge: disableFullPageSwipe(isUserLoggedIn: authBloc.state.isLoggedIn, state: thunderBloc.state, isFeedPage: true),
-      builder: (context) => MultiBlocProvider(
-        providers: [
-          BlocProvider.value(value: accountBloc),
-          BlocProvider.value(value: authBloc),
-          BlocProvider.value(value: thunderBloc),
-        ],
-        child: CommunityPage(communityId: communityId),
-      ),
-    ),
-  );
+  navigateToCommunityPage(context, communityId: communityId);
```

Comment:
Do we even need these wrapper functions any more, or could we just put `navigateToCommunityPage` right in the tap handler?